includes/class-theme-base.php

```diff
@@ -547,32 +564,51 @@ function post() {
 
 	/**
 	 * @return WPLib_Post_Base
-	 *
-	 * @future Make work for non-posts?
 	 */
 	function item() {
 
-		return $this->has_posts() 
-			? WPLib_Posts::make_new_item( $this->post() ) 
-			: new WPLib_Post_Default( null );
+		$item = null;
+		if( $this->has_posts() ) {
+			$_post = $this->post();
+			$item  = WPLib_Posts::make_new_item( $_post, "list_owner=" );
```

Comment:
@sarutole I want to understand why these 4 lines are needed.  When will `$item` be false?  Why is `list_owner` set to empty string